Makefile

```diff
@@ -59,6 +59,39 @@ E2E_TEST_CONFIG ?=
 
 LDFLAGS = -ldflags "-s -w -X sigs.k8s.io/node-feature-discovery/pkg/version.version=$(VERSION) -X sigs.k8s.io/node-feature-discovery/source.pathPrefix=$(HOSTMOUNT_PREFIX)"
 
+## multi-arch build
+# required to enable buildx
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+# build with buildx
+PLATFORMS ?= linux/amd64,linux/arm64
+PROGRESS   = auto
+# buildx keeps the build in the cache no default output
+# use --push as OUTPUT to push to registry
+# or --load to have it available as an image
+OUTPUT =
```

Comment:
Should/could we get rid of this `OUTPUT` variable, looks kinda odd and stray. What about just appending `--pull`/`--push` in the corresponding builder commands, e.g.

```
$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_BUILD_ARGS_FULL) --pull
```